lib/kwtool_web/templates/keyword/_keyword-result.html.eex

```diff
@@ -0,0 +1,46 @@
+<%= if length(@keyword_results) > 0 do %>
```

Comment:
I think using the function would make the condition easier to read
```suggestion
<%= if Enum.any?(@keyword_results) do %>
```

Reply:
Updated in https://github.com/byhbt/kwtool/pull/44/commits/213a5dfa7b88ebb6f5e12f03c5b2ea849a9b56ca